.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -1,24 +1,27 @@
 ---
-name: Feature request
-about: Request a new feature or improvement in this library
+name: Library-specific Feature Request
+about: Suggest an idea to make this library better.
 title: ''
-labels: enhancement
+labels: enhancement, triage
 assignees: ''
 
 ---
 
-<!--
-This form is for feature requests for this library only.
-For general, language-agnostic questions and troubleshooting, please ask/look for answers [here](https://groups.google.com/g/adwords-api).
+<!-- PLEASE READ
+Make sure you've carefully reviewed the documentation for this library to make sure your feature request is not already supported: https://developers.google.com/google-ads/api/docs/client-libs/php.
 
-Please check existing issues to see if your feature request has already been identified before creating a new issue. We encourage you to add any relevant information to the existing issue if one already exists.
+For requests that are related to the Google Ads API itself and not specific to this library, please reach out to one of our support channels: https://developers.google.com/google-ads/api/support.
+
+Please check existing issues to see if your feature request has already been reported before creating a new issue. We encourage you to add any relevant information to the existing issue if one already exists.
 -->
 
-**Describe the problem you are trying to solve**
+[DO NOT INCLUDE ANY PERSONAL OR SENSITIVE DATA - MAKE SURE TO REDACT CONTENT WHEN NECESSARY]
```

Comment:
Do you think this is needed for FRs?

Reply:
Not for all FRs for sure but some may include some content describing how something works today (e.g., a request, a response, a code snippet)

Reply:
+1. Better safe than sorry. I don't think it's harmful to put this sentence here.